### PR TITLE
Integrate video

### PR DESCRIPTION
**Pull request by kinsaurralde**
- Added VideoPlayer class which takes a list of youtube video ids and plays them in order
- calling player.playVideos() with a list of youtube video ids will bring the youtube player in front of the map
- when video is done, youtube player will be hidden

### CODE REVIEW

File: src/main/webapp/script.js
Comment (panerorenn9541) [inline src/main/webapp/script.js]: I'm not 100% sure, but you might want the comments above each function to be formatted with 
/**
 *
 */

**Review by panerorenn9541 — APPROVED** [review]
Looks good

**Review by alsile — APPROVED** [review]
lgtm

**Review by huzcn — APPROVED** [review]
LGTM on my side. Please wait for the review from Yang.

**Review by v01dstar — APPROVED** [review]
LGTM

**Pull request merged**
